core/kits/manager.php

```diff
@@ -97,7 +97,7 @@ public function create( array $kit_data = [], array $kit_meta_data = [] ) {
 		return $kit->get_id();
 	}
 
-	private function create_default() {
+	public function create_default() {
```

Comment:
I think `create_default` should also update the active kit option.

Reply:
Replaced private to public/
About the suggestion I will wait with it until I will see if this PR will be merged since today the exist code always create kit automatically when deleted.